source/index.html.md

```diff
@@ -540,26 +554,123 @@ Exactly one payment method must be used.
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential has already been stored and is now
+    being used.
+    <br />
+    Default:
+    <ul>
+      <li><code>using</code>, if <code>initiator</code> is <code>merchant</code>
+      (<code>storing</code> is invalid),</li>
+      <li><code>storing</code>, if the authorization is first-in-series.</li>
+    </ul>
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the authorization. An authorization is initiated by the
+    cardholder if the cardholder decided the transaction should be created.
+    This is regardless of whether stored payment credential is being used.
+    <br />
+    For compliance reasons there should be a previous approved transaction (for
+    the cardholder and the merchant) marked with <code>storing</code> before
+    <code>initiator</code> may be <code>merchant</code>.
+    <br />
+    Default:
+    <ul>
+      <li><code>merchant</code>, if the authorization is
+      subsequent-in-subsequent (<code>cardholder</code> is invalid),</li>
+      <li><code>cardholder</code>, otherwise.</li>
+    </ul>
+
+    <div class="type">Optional</div>
+  </dd>
+
   <dt>ip
     <span class="type">[0-9.a-fA-F:]{3,45}</span>
   </dt>
   <dd>
     Cardholder's IP address. It must be a valid v4 or v6 address.
     <div class="type">Optional<br></div>
   </dd>
-  <dt>recurring
-    <span class="type">(true|false)</span>
+
+  <dt>reference
+    <span class="type">[\x20-\x7E]{1,30}
+      <a target="_blank" href= "http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
   </dt>
   <dd>
-    Must be <code>true</code> for recurring payments.
+    A reference to an external object, such as an order number.
     <div class="type">Optional</div>
   </dd>
+
+  <dt>
+    series[type]
+    <span class="type">(<code>recurring</code>|<code>unscheduled</code>)</span>
+  </dt>
+  <dd>
+    Indicate the type of series.
+    <br />
+    <code>recurring</code>: A series of transactions where the cardholder has
+    explicitly agreed that the merchant will repeatedly charge the cardholder at
+    regular, predetermined intervals that may not exceed 1 year.
+    <br />
+    <code>unscheduled</code>: A series of transactions where the cardholder has
+    explicitly agreed that the merchant will repeatedly charge the cardholder at
+    unknown times, e.g. based on cardholder usage.
+    <br />
+    Although the amount may vary among transactions in any series, the amount
+    of any subsequent-in-series transaction can not exceed the amount of the
+    authentication of the first-in-series authorization.
```

Comment:
```suggestion
    of any subsequent-in-series transaction may not exceed the amount of the
    <b>authentication</b> of the first-in-series authorization.
```

Reply:
😄 I actually wrote `may` first, but ... oh well ... https://english.stackexchange.com/questions/56065/must-not-or-may-not-which-is-the-most-correct

We should probably use "RFC style" wording — and reference https://tools.ietf.org/html/rfc2119

Reply:
I also like "must", perhaps better than "may", however, I think "cannot" isn't quite correct :slightly_smiling_face:

Reply:
Are we going to enforce this? Can not (can) imply that we return an error if you try. Must/may not more has a "it is on you to secure".

Reply:
@ad-clearhaus Although I don't agree with your interpretation of the words, I think whether or not we will enforce the rule is a very relevant question.

Reply:
We cannot (`:tøhø:`) enforce it because we are blind about the authentication amount.

It is a compliance-wise "it wrong if you do" — if scheme or issuer declines, I have no clue, but they ought to decline.

Perhaps we should move this entire section into the guideline?

Reply:
Maybe this should indeed be moved to guidelines.
We are not going to enforce it. The issuers should enforce it and decline. Which is fine. Then we can return that to the PSP's. As it stands we have no transaction evidence pointing towards the issuers actually declining.
To me, this issue seems like something where we need to see over time how the ecosystem reacts.

But I still think, in guidelines or not, that _cannot_ should be replaced by _must_ ;)

Reply:
Resolved by 8415d33.